src/governance.spec.ts

```diff
@@ -535,4 +535,76 @@ describe("GovernanceCanister.claimOrRefreshNeuron", () => {
       expect(call).rejects.toThrow(new GovernanceError(error));
     });
   });
+
+  describe("GovernanceCanister.startDissolving", () => {
+    it("successfully starts dissolving process", async () => {
+      const neuronId = BigInt(10);
+      const serviceResponse: ManageNeuronResponse = {
```

Comment:
Is `serviceResponse` needed for the test?

Reply:
We need some kind of response, undefined is not enough, the return is checked in the helper `manageNeuron`.

```
  if (!response) {
    throw new GovernanceError({
      error_message: "Error updating neuron",
      error_type: 0,
    });
  }

  if ("Error" in response) {
    throw new GovernanceError(response.Error);
  }
```

Since we need something, it makes sense it's similar to the real response we get in successful case.